analyzer/codechecker_analyzer/analyzers/clangtidy/analyzer.py

```diff
@@ -128,12 +130,28 @@ def get_diagtool_bin():
     if not clang_bin:
         return None
 
+    path_env = os.environ['PATH'].split(os.pathsep)
+    clang_path = Path(clang_bin)
+
+    if clang_path.resolve().name == 'ccache':
```

Comment:
Is the problem only defined to ccache?
Shouldnt you check if clang_bin is a symlink?